docker/session-manager/env-dev

```diff
@@ -0,0 +1,10 @@
+API_URL=https://localhost:8080/functions-app
```

Comment:
Session manager is not reaching fn-app right now, this configuration fixed the issue

```suggestion
API_URL=http://host.docker.internal:7001/api/v1
```